docs/developers/tutorials/crowdfunding-p1.md

```diff
@@ -72,13 +72,13 @@ The source code of each smart contract requires its own folder. You'll need to c
 ```bash
 mkdir -p ~/MultiversX/SmartContracts
 cd ~/MultiversX/SmartContracts
-mxpy contract new crowdfunding --template adder
+mxpy contract new crowdfunding --template empty
```

Comment:
The build is currently failing, but we can fix it with something like this (after invoking `mxpy contract new`):

```
# Temporary workaround, will be fixed soon, when "mxpy" integrates the new templating facility (of "sc-meta").
mv ./crowdfunding/src/empty.rs ./crowdfunding/src/crowdfunding.rs 
```

Reply:
it was a problem with the tests from rust, I fixed it